lib/MusicBrainz/Server/Entity/Artwork.pm

```diff
@@ -101,22 +101,26 @@ sub filename
 sub image { my $self = shift; return $self->_url_prefix . "." . $self->suffix; }
 sub small_thumbnail { my $self = shift; return $self->_url_prefix . "-250.jpg"; }
 sub large_thumbnail { my $self = shift; return $self->_url_prefix . "-500.jpg"; }
+sub huge_thumbnail { my $self = shift; return $self->_url_prefix . "-1200.jpg"; }
```

Comment:
"huge" is probably fine as an internal-only name, though for reference we did decide against using that in the API (index.json) and just opted for 250/500/1200 as names (with "small" and "large" kept for backwards compatibility).

Reply:
We could also rename these to thumbnail_250 etc, but it might break some scripts or something if we change it for the `data-` attributes.

Reply:
Yeah, I'm fine with it how it is, just wanted to point it out so we don't start using it in public pages/APIs in the future.